### PR TITLE
[APP-3046] Post logout redirect URL update.

### PR DESCRIPTION
**Pull request by maxhorowitz**
Depending on auth provider, we need to populate post logout redirect URL with different query params.

### CODE REVIEW

File: web/auth0.go [hunk 1]
```diff
@@ -106,10 +107,18 @@ func InstallFusionAuth(
...
 	logger golog.Logger,
 ) (io.Closer, error) {
+	overwrite := AuthProviderConfig{
+		Domain:                config.Domain,
+		ClientID:              config.ClientID,
+		Secret:                config.Secret,
+		BaseURL:               config.BaseURL,
+		PostLogoutRedirectURL: "post_logout_redirect_uri",
+		EnableTest:            config.EnableTest,
+	}
```
Comment (ohEmily) [inline web/auth0.go:117]: I think you can instead just do `config.PostLogoutRedirectURL = "post_logout_redirect_uri"`, no?

And then in `InstallAuth0`, add `config.PostLogoutRedirectURL = "returnTo"`.

Maybe I'm missing something?

Comment (ohEmily) [reply web/auth0.go:117]: Then you could remove this change if I understand correctly -- https://github.com/viamrobotics/app/pull/3982/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R431-R433

Comment (maxhorowitz) [reply web/auth0.go:117]: yes

Comment (maxhorowitz) [reply web/auth0.go:117]: completely

**Review by ohEmily — APPROVED** [review]
LGTM, thank you!

**Pull request merged**